api/app/services/prometheus_client.py

```
@@ -0,0 +1,200 @@
+        """
+        promql = "bisq_queries_total"
+
+        result = await self.query(promql)
+        if not result or not result.get("data", {}).get("result"):
+            logger.debug("No query count data available in Prometheus")
+            return None
+
+        try:
+            value = int(float(result["data"]["result"][0]["value"][1]))
+            logger.info(f"Retrieved total query count from Prometheus: {value}")
+            return value
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)